doc/s3.md

```diff
@@ -1,26 +1,40 @@
 # TL;DR
 
-Create and describe s3 buckets
+Create, describe and interact with s3 buckets
 
 ## Use
 
 ### list or ls
+Lists objects in bucketPath if provided, otherwise lists available buckets
 ```
   gen3 s3 list [bucketPath]
 ```
-Lists objects in bucketPath if provided, otherwise lists available buckets
-  - [bucketPath] - s3 path to list
+Options:
+  - [bucketPath]: s3 path to list (e.g. mybucket/subfolder/)
 
-### new 
+### create 
+Creates a new s3 bucket if it doesn't already exist
 ```
-  gen3 s3 new <bucketName>
+  gen3 s3 create <bucketName> [--add-cloudtrail]
 ```
-Creates a new s3 bucket if it doesn't already exist
-  - bucketName - name of bucket to create
+Options:
+  - bucketName: name of bucket to create
+  - --add-cloudwatch: add bucket logging to cloudtrail
```

Comment:
Typo?  Should be `--add-cloudtrail`.  Also - will this add the bucket to an existing trail, or create a new trail for the bucket?